Source/ACE.Server/WorldObjects/Player_Database.cs

```diff
@@ -56,7 +74,9 @@ public void RushNextPlayerSave(int seconds)
         /// </summary>
         public void SavePlayerToDatabase()
         {
-            if (CharacterChangesDetected)
+            if (DoNotSave) { return; }
```

Comment:
The check shouldn't be here, when it already exists at the WorldObject level

Reply:
Yeah this ends up being a catch all I guess. If I never set changes detected to true it should never run anyways, is that what you are referring to?

Reply:
removed